oem/ibm/libpldmresponder/file_io_type_pcie.cpp

```diff
@@ -206,10 +206,11 @@ void PCIeInfoHandler::getMexObjects()
     // get the in memory cached copy of mex objects
     auto savedObjs = pldm::serialize::Serialize::getSerialize().getSavedObjs();
 
-    // Find the PCIE slots & PCIeAdapters & Connecters & their location codes
+    // Find the PCIE slots & PCIE logical slots & PCIeAdapters & Connecters &
+    // their location codes
     std::set<uint16_t> neededEntityTypes = {PLDM_ENTITY_SLOT, PLDM_ENTITY_CARD,
                                             PLDM_ENTITY_CONNECTOR,
-                                            PLDM_ENTITY_SYSTEM_CHASSIS};
+                                            PLDM_ENTITY_SYSTEM_CHASSIS, 32954};
```

Comment:
magic number?

Reply:
This is the logical slot entity Id , we have used the same at multiple places so adding the same to avoid confusion

Reply:
why can't we add ENUM string instead of magic number? 
and readability is also problem incase of magic number, if we don't have any mechanism yet then need to think to avoid magic numbers in code.

Reply:
we can change it to 0x8000 | PLDM_ENTITY_SLOT

Reply:
Done